src/components/common/header/Header.module.css

```diff
@@ -0,0 +1,41 @@
+/* Scoped header styles: dark-blue background with gold bottom border */
+.header {
+  width: 100%;
+  background: var(--color-background-dark-blue);
+  border-bottom: 4px solid var(--color-gold);
+  color: var(--color-typography-primary-invert);
```

Comment:
shouldn't the header have a fixed position so it stays at the top as you scroll down